tests/api/testdata/db_filler.py

```diff
@@ -248,9 +269,9 @@ def _choose_fk_field_value(self, field: Field, fk_data: list):
         """Choose a random fk value for the specified field"""
         if isinstance(field.f_type, ForeignKey):
             fk_class_name = field.f_type.fk_link.__name__
-            if fk_class_name in self._available_fkeys:
+            if available_fkeys := self._available_fkeys.get(fk_class_name, None):
```

Comment:
please add a comment to explain why was it required

Reply:
`if fk_vals := ...`

Reply:
Tried to add decent explanation here